README.md

```diff
@@ -0,0 +1,5 @@
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Please use the [Microverse's README templane](https://github.com/microverseinc/readme-template).

Reply:
- [ ] I think you can improve this message by adding a short explanation, a "why", the requested change needed to be made. This will make your comment more professional. :blush:

Reply:
added `this template is more professional and complete.`